src/photo.ts

```diff
@@ -0,0 +1,10 @@
+export default interface photo {
+    id: number;
+    sol: number;
+    camera: { id: number;
+        name: string;
+        rover_id: number;
+        full_name: string;};
+    img_src: string;

```

Comment:
Nit: Styling nested objects like this makes them a little more readable/uniform

```suggestion
    camera: { 
        id: number;
        name: string;
        rover_id: number;
        full_name: string;
    };
    img_src: string;

```